samples/com.google.android.gms/play-services-fitness/BasicSensorsApi/MainActivity.cs

```diff
@@ -74,7 +74,7 @@ void buildFitnessClient()
             // Create the Google API Client
             mClient = new Android.Gms.Common.Apis.GoogleApiClient.Builder (this)
                 .AddApi(Android.Gms.Fitness.FitnessClass.SENSORS_API)
-                .AddScope(new Android.Gms.Common.Apis.Scope(Android.Gms.Common.Scopes.FitnessLocationRead))
+                //.AddScope(new Android.Gms.Common.Apis.Scope(Android.Gms.Common.Scopes.FitnessLocationRead))
```

Comment:
Is this removal needed because of an API change/break?

Why comment out and not just remove?